README.md

```diff
@@ -132,48 +132,87 @@ required when they are placed in the environment file.
+.irodsA file directly:
 
-Note, in the `pam_password` case, this involves sending the cleartext password
-to the server (SSL must be enabled!) and then writing the scrambled token that
-is returned from the transaction.
+```bash
+$ echo '{ "irods_user_name":"rods", ...  }'> ~/.irods/irods_environment.json
+$ python -c "import irods.client_init, getpass
+irods.client_init.write_native_irodsA_file(getpass.getpass('Enter iRODS password -> '))"
+```
 
-If an .irodsA file exists already, it will be overwritten by default; however, if these functions'
+If an .irodsA file already exists, it will be overwritten by default; however, if these functions'
 overwrite parameter is set to `False`, an exception of type `irods.client_init.irodsA_already_exists`
-will be raised to indicate the older .irodsA file is present.
+will be raised to warn of an older .irodsA file that might otherwise be overwritten.
 
-For the `pam_password` authentication scheme, we must first ensure an `irods_environment.json` file exists in the 
-client environment (necessary for establishing SSL/TLS connection parameters as well as obtaining a PAM token from the server after connecting)
-and then make the call to write .irodsA using the Bash commands:
+Equivalently, we can issue the following command.
 
 ```bash
-$ cat > ~/.irods/irods_environment.json << EOF
-{
-  "irods_user_name":"rods",
-  "irods_host":"server-hostname",
-  ...  [all other connection settings, including SSL parameters, needed for communication with iRODS] ...
-}
-EOF
-$ python -c "import irods.client_init as iinit; iinit.write_pam_credentials_to_secrets_file(pam_cleartext_password)"
+$ prc_write_irodsA.py native <<<"${MY_CURRENT_IRODS_PASSWORD}"
 ```
 
-PAM logins
-----------
+The redirect may be left off, in which case the user is prompted for the iRODS password
+and echo of the keyboard input will be suppressed.  (Regardless which technique is used,
+no password will be visible on the terminal during or after input.)
+
+For the `pam_password` scheme, typically SSL/TLS must first be enabled to avoid sending data related
+to the password - or even sending the raw password itself - over a network connection in the clear.
+
+Thus, for `pam_password` authentication to work well, we should first ensure when setting up the
+client environment that the `irods_environment.json` file includes the appropriate
+SSL/TLS connection parameters.  If present, `iinit` can be used to verify this condition is fulfilled,
+as of course its invocation would create a valid .irodsA from merely prompting the user for their
+PAM password
+
+But if we wish to use the Python client for this purpose instead, we can run:
+
+```python
+irods.client_init.write_pam_irodsA_file(getpass.getpass('Enter current PAM password -> '))
+```
+
+Or from the Bash command shell, we simply run:
+
+```bash
+$ prc_write_irodsA.py pam_password <<<"${MY_CURRENT_PAM_PASSWORD}"
+```
+
+again leaving out the redirection if password prompting is preferable.
```

Comment:
Maybe show what that looks like, just in case?

Reply:
Right, sorry. Wasn't sure if it would be helpful to show with the redirection left out so the reader can see the prompt. Now that I'm looking at it again, that actually doesn't seem very useful. Feel free to just resolve this. More help is (usually) better, though, so, up to you.

Reply:
Resolving, I guess it's clear enough for now...